packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -157,6 +158,18 @@ async function main(): Promise<number | void> {
     }
 
     if (adapterProviders.some(isDriverAdapterProviderLabel)) {
+      // Locally, running D1 tests accumulates a lot of data in the .wrangler directory.
+      // Because we cannot reset the database contents programmatically at the moment,
+      // deleting it is the easy way
+      // It makes local tests consistently fast and clean
+      try {
+        fs.rmdirSync(path.join(__dirname, '..', '..', '.wrangler'), { recursive: true })
+      } catch (e) {
+        if (e.code !== 'ENOENT') {
+          console.warn('Deleting the .wrangler directory failed with:', e)
+        }
+      }
```

Comment:
```suggestion
      fs.rmSync(path.join(__dirname, '..', '..', '.wrangler'), { recursive: true, force: true })
```

`rm -rf` equivalent that does not errors on non existent dir.